www-apps/miniflux/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>me@concord.sh</email>
```

Comment:
This doesn't match your git author/name in other metadata.xmls in this PR.

Reply:
It will be concord<at>g.o once recruitment is finished :)

Reply:
I figured :)

But the name part is what I meant!

Reply:
> But the name part is what I meant!

Good catch. All fixed. :)